web/package.json

```diff
@@ -37,6 +39,7 @@
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
     "react-icons": "^4.2.0",
+    "react-is": "^17.0.1",
```

Comment:
Do we need to add this?

Reply:
@asasmith im scared to remove it after the build fiasco

Reply:
I would remove them one at a time and see if there is a specific one that breaks the build. I'm fairly certain none of them are necessary.